content/blog/2020-07-29-openlayer-one-year/index.fr.md

```diff
@@ -0,0 +1,54 @@
+---
+title: OpenLayer, 1 an plus tard
+author: David Beauchemin
+date: '2020-07-29'
+slug: podcast
+type: post
+categories: ["podcast"]
+tags: []
+description: "Retour sur la première année d'existence d'OpenLayer"
+featured: "OpenLayer_YoutubeBanner.jpg"
+featuredpath: "img/headers/"
+---
+
+Cela fait maintenant 1 an que j’ai lancé OpenLayer, mon projet de podcast vidéo sur l'IA. 32 épisodes plus tard, beaucoup de choses se sont dites derrière le micro. Ce texte débute par une présentation de mon voyage dans cette aventure et je termine avec un survol de certains moments forts des diverses discussions que j’ai eu la chance d’avoir avec les invités.
+
+## Le projet
+
+D’abord, c’est quoi OpenLayer?
+
+Ça se veut un lieu ou une *myriade* de personnes ayant des profils aux antipodes l’un des autres, mais gravitant autour de cette *bébitte* qu’est l'IA. De l’expert en recherche en apprentissage automatique en passant par l’avocat en propriété intellectuelle jusqu’aux ressources humaines ou la santé. Nous avons eu droit à beaucoup d’informations sur cet univers selon différents acteurs du milieu.
+
+Cette idée d'avoir un podcast vidéo m’est venue il y a environ 2-3 ans en écoutant Mike Ward sous écoute (j’ai écouté l’ensemble des épisodes ou à peu près #BRAG). Je me suis dit, il y a quelque chose d’intéressant à faire avec cela. Par la suite, les choses se sont mises en branle lorsque j’ai quitté mon appartement d’étudiant laid sur l'avenue Myrand. Un mur « blanc » vieillis par le soleil et les années en arrière-plan accompagner du son de mon voisin qui gueule (littéralement) en gammant c’est pas l’arrière-plan le plus excitant mettons.
+
+### Les débuts
+
+*Fast-forward* à l’été 2019, j’ai débuté en invitant un collègue et ami, [Nicolas Garneau](https://youtu.be/x9Zo-F0PWQE), pour commencer avec un invité plus « facile » à interagir avec, puisque je le connais bien. C’est un épisode intéressant puisqu’on aborde différents aspects personnels sur sa vie, mais aussi des aspects plus techniques comme la qualité du code en recherche et ce qui se veut être une introduction à la reproductibilité en science.
+
+### L’évolution
+
+S’en ait suivie plusieurs expérimentations dans les épisodes suivants : le décor à utiliser, la configuration de la console de son, positionnement des micros et de la caméra, le contrôle de la lumière dans l’environnement et de mes chats dans l’arrière-plan. Plusieurs points ont été corrigés, mais d’autres restent à améliorer continuellement.
```

Comment:
Je sais pas trop comment gérer le "S'en ait suivie", mais il semble (à ma surprise) que ce ne soit pas accepté comme façon de parler/écrire : https://plus.lesoir.be/128456/article/2017-12-08/sensuivre-ou-sen-suivre

Reply:
Intéressant.